start-workflow-from-archive/README.md

```diff
@@ -0,0 +1,60 @@
+This script allow you to start a workflow on an archived event.
+
+Install:
+```
+$ pip install -r requirments.txt
```

Comment:
Since this script is Python 3 only, this is probably `pip3` unless you already have Python 3 as default. You should also mention that you require Python 3